blueprint/introduction/json-api.apib

```diff
@@ -47,7 +47,8 @@ For some models, you can also search based on the query text:
 
 You can choose how much of a resource to receive by specifying
 pagination parameters. Pagination is supported via `limit` and
-`offset`. Resources are paginated in groups of `20` by default.
+`offset`. Resources are paginated in groups of `10` by default and can be
+increased to a maximum of `20`.
```

Comment:
Probably best to mention `libraryEntries`'s limit is `400`

Reply:
It seems the limit is actually `500` according to [this](https://github.com/hummingbird-me/hummingbird-server/blob/a45e7e567138ed5e6ff26b1d10fb0efa48213fe5/lib/library_paginator.rb). I've also noticed several resources have [no limit](https://github.com/hummingbird-me/hummingbird-server/blob/a45e7e567138ed5e6ff26b1d10fb0efa48213fe5/lib/unlimited_paginator.rb), so maybe it would be better to mention it per resource if the maximum page limit is not the default (`20`).